src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationProcessingServiceImpl.java

```diff
@@ -35,35 +36,49 @@ public void processConcurEventNotifications() {
 
             ConcurReport concurReport = concurReportsService.extractConcurReport(concurEventNotification.getObjectURI());
+
             if (concurReport != null) {
-                if (concurReport.getAccountInfos() != null) {
-                    LOG.info("Validate account info: " + KFSConstants.NEWLINE);
-                    for (ConcurAccountInfo concurAccountInfo : concurReport.getAccountInfos()) {
-                        LOG.info(concurAccountInfo.toString());
-                        validationResult = concurAccountValidationService.validateConcurAccountInfo(concurAccountInfo);
-                        LOG.info("Validation Result: " + validationResult.isValid() + ", validation messages: " + validationResult.getErrorMessagesAsOneFormattedString());
+                LOG.info("Concur report status code from Concur: " + concurReport.getConcurStatusCode() + ", workflow URI: " + concurReport.getWorkflowURI());
+                if (ConcurUtils.isConcurReportStatusAwaitingExternalValidation(concurReport.getConcurStatusCode())) {
+                    LOG.info("Validate report accounting info");
+                    validateAccountInfo(validationResult, concurReport);
+                    
+                    LOG.info("Update report status in Concur");
+                    concurReportsService.updateExpenseReportStatusInConcur(concurReport.getWorkflowURI(), validationResult);
                     }
-                } else {
-                    LOG.info("No account info present.");
-                    validationResult.addMessage(configurationService.getPropertyValueAsString(ConcurKeyConstants.CONCUR_ACCOUNT_INFO_IS_REQUIRED));
+                else{
+                    LOG.info("Concur Report not in Awaiting External Validation status");
+                    validationResult.addMessage(configurationService.getPropertyValueAsString(ConcurKeyConstants.INCORRECT_CONCUR_STATUS_CODE));
                 }
                 
-                LOG.info("Update report status in Concur");
-                concurReportsService.updateExpenseReportStatusInConcur(concurReport.getWorkflowURI(), validationResult);  
-                LOG.info("Update ConcurEventNotification flags and validationMessage in KFS database");
-                concurEventNotificationService.updateConcurEventNotificationFlagsAndValidationMessage(concurEventNotification, false, true, validationResult.isValid(), validationResult.getErrorMessagesAsOneFormattedString());
+                LOG.info("Update ConcurEventNotification flags and validationMessage in KFS database: " + validationResult.isValid() + "," + validationResult.getErrorMessagesAsOneFormattedString());
+                concurEventNotificationService.updateConcurEventNotificationFlagsAndValidationMessage(concurEventNotification, false, true, validationResult.isValid(), validationResult.getErrorMessagesAsOneFormattedString());           
```

Comment:
Using local variables with descriptive names would give clarity as to what "false" and "true" are representing in this method call.